### ISSUE TITLE
[Feature Request] Always add `color-scheme: dark`

### ISSUE THREAD

**Issue by KaKi87** [darkreader/darkreader#10992]
### Prerequisites

- [X] I [searched for any existing report](https://github.com/darkreader/darkreader/issues?q=is%3Aissue) about this feature request to avoid opening a duplicate.

### Is this feature request related to a problem?

Yes

### Feature Request Description

Too many websites lack the [`color-scheme: dark`](https://developer.mozilla.org/en-US/docs/Web/CSS/color-scheme) CSS property despite being dark, leaving the scrollbar light.

So, I would like to suggest that, as long as Dark Reader is enabled, it injects this property to the page, even when the site is in the global dark list or when it uses the `darkreader-lock` meta tag.

### Screenshots

For example, here's a screenshot of a page from the very first website in the global dark list :

![Screenshot from 2023-03-28 22-41-37](https://user-images.githubusercontent.com/21284089/228362033-22ed268d-6583-456d-aa55-76b2a2b0dbcd.png)

Dark theme, but light scrollbar.

### Additional Context

None

---

Thanks

**Comment by paschun** [comment]
Yeah I'm honestly shocked darkreader doesn't already do this. Many sites already have dark theme toggles that are implemented via `color-scheme` and `light-dark()` CSS. It would make much more sense to simply use the dark mode theme that the website has already created. Which can be as simple as this single line of CSS.
